libraries/bsg_manycore_eva.cpp

```diff
@@ -382,7 +382,13 @@ static uint32_t default_get_dram_bitwidth(const hb_mc_manycore_t *mc)
         if (hb_mc_manycore_dram_is_enabled(mc)) {
```

Comment:
just don't check if dram is enabled/disabled and default to `hb_mc_config_get_vcache_bitwidth_data_addr`